modules/storages/app/components/storages/project_storages/project_mapping/row_component.rb

```diff
@@ -0,0 +1,34 @@
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2024 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+# Purpose: Defines how to format the components within a table row of Projects
+# associated with a Storage
+module Storages::ProjectStorages::ProjectMapping
```

Comment:
Given the new namespacing structure in https://github.com/opf/openproject/pull/16018 I think the following would be more fitting- as we essentially can drop the "project_mapping" naming across the board
```suggestion
module Storages::ProjectStorages
```

Reply:
i.e. we'd have

1. `Storages::ProjectStorages::ProjectRowComponent` - for  projects
2. `Storages::ProjectStorages::StorageRowComponent` - for  storages _(in the future when the [current list is primerised](https://qa.openproject-edge.com/projects/demo-project/settings/project_storages/external_file_storages))_

Reply:
I'd rather go with another namespace here. It's also nice if the files are grouped (not only by name but by a folder). What do you think about
`Storages::ProjectStorages::Project::RowComponent` (and `Storages::ProjectStorages::Project::TableComponent`) and keeping the other files as is?

Reply:
That also works- +1 on folder naming, can be further reused 👍🏾